### ISSUE TITLE
[Question]: Why does model.__call__ return the loss too?

### ISSUE THREAD

**Issue by dclong** [huggingface/transformers#3144]
# ❓ Questions & Help

In PyTorch, model.__call__ returns the output tensor and users have to call a loss function to get the loss. I wonder why models in transformers doesn't follow this convention? Any specific reason?
